hops/util.py

```diff
@@ -72,7 +75,7 @@ def _get_hopsworks_rest_endpoint():
         The hopsworks REST endpoint for making requests to the REST API
 
     """
-    return os.environ[constants.ENV_VARIABLES.REST_ENDPOINT_END_VAR]
+    return 'https://' + os.environ[constants.ENV_VARIABLES.REST_ENDPOINT_END_VAR]
```

Comment:
In Hopsworks the environment variable already contain `https://`

Reply:
Ok, now I get what you meant by "in hopsworks", I fixed it.